vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SequenceManagerTest.java

```diff
@@ -0,0 +1,113 @@
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.diging.vspace.core.data.SequenceRepository;
+import edu.asu.diging.vspace.core.data.SlideRepository;
+import edu.asu.diging.vspace.core.factory.impl.SequenceFactory;
+import edu.asu.diging.vspace.core.model.ISequence;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.model.impl.Module;
+import edu.asu.diging.vspace.core.model.impl.Sequence;
+import edu.asu.diging.vspace.core.model.impl.Slide;
+import edu.asu.diging.vspace.web.staff.forms.SequenceForm;
+
+public class SequenceManagerTest {
+
+    @Mock
+    private SequenceRepository mockSequenceRepo;
+
+    @Mock
+    private ModuleManager moduleManager;
+    
+    @Mock
+    private SlideManager slideManager;
+    
+    @Mock
+    private SlideRepository slideRepo;
+    
+    @Mock
+    private SequenceFactory sequenceFactory;
+    
+    @InjectMocks
+    private SequenceManager sequenceManagerToTest = new SequenceManager();
+    
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void test_getSequence_idExists() {
+        Sequence newSequence = new Sequence();
+        newSequence.setId("sequence1");
+        Optional<Sequence> mockSequence = Optional.of(newSequence);
+        Mockito.when(mockSequenceRepo.findById(newSequence.getId())).thenReturn(mockSequence);
+
+        ISequence actualSequence = sequenceManagerToTest.getSequence(newSequence.getId());
+        Assert.assertEquals(newSequence.getId(), actualSequence.getId());
+        Assert.assertEquals(newSequence.getName(), actualSequence.getName());
+    }
+    
+    @Test
+    public void test_getSequence_idNotExists() throws Exception {
+        Mockito.when(mockSequenceRepo.findById("a non existing id string")).thenReturn(Optional.empty());
+        Assert.assertNull(sequenceManagerToTest.getSequence("a non existing id string"));
+    }
+    
+    @Test
+    public void test_storeSequence_success() {        
+        Module module = new Module();
+        module.setId("module1");
+             
+        SequenceForm sequenceForm = new SequenceForm();
+        sequenceForm.setName("seq1");
+        sequenceForm.setDescription("sample description");
+              
+        String slideId1 = "slide1";
+        String slideId2 = "slide2";
+        List<String> slideIds = new ArrayList<>();
+        slideIds.add(slideId1);
+        slideIds.add(slideId2);
+        sequenceForm.setOrderedSlides(slideIds);
+        
+        ISlide slide1 = new Slide();
+        slide1.setId("slide1");       
+        ISlide slide2 = new Slide();
+        slide2.setId("slide2");
+ 
+        List<ISlide> slides = new ArrayList<>();
+        slides.add(slide1);
+        slides.add(slide2);
```

Comment:
I believe there is a ```Arrays.asList()``` method that just takes several objects and returns a list of them. This would turn these three lines into a one-liner.